config/jobs/kubernetes-sigs/sig-windows/release-1.27-windows-presubmits.yaml

```diff
@@ -0,0 +1,40 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-capz-windows-1-27
+    decorate: true
+    always_run: false
+    optional: true
+    run_if_changed: 'azure.*\.go$|.*windows\.go$|test/e2e/windows/.*'
+    path_alias: k8s.io/kubernetes
+    branches:
+      - release-1.27
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
+      preset-capz-windows-2022: "true"
+      preset-capz-containerd-1-7-latest: "true"
+      preset-capz-windows-common-pull: "true"
+    extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-azure
```

Comment:
do we need a cloud-provider for the PR job too?

Reply:
Yea, we do.
I think the presubmit jobs for the other releases might be broken... let me fix this.